omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/VisitTypeController1_9Test.java

```diff
@@ -134,4 +139,15 @@ public void shouldSearchAndReturnAListOfVisitTypesMatchingTheQueryStringExcludin
 		
 	}
 	
+	@Test
+	public void shouldGetAVisitTypeByUuid() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + getUuid());
+		SimpleObject result = deserialize(handle(req));
+		
+		VisitType visitType = service.getVisitTypeByUuid(getUuid());
+		assertNotNull(result);
```

Comment:
Do we really need the above assert?

Reply:
After stepping back and thought about the assert, I realized we do not need it and I did the needful!

Reply:
After stepping back and thought about the assert, I realized we do not need it and I did the needful!